src/main/java/org/manifold/compiler/front/Main.java

```diff
@@ -241,6 +257,26 @@ public void reportContextSensitivity(@NotNull Parser recognizer, @NotNull DFA df
   }
 }
 
+class ImportVisitor extends ManifoldBaseVisitor<Void> {
```

Comment:
Didn't we have a plan to remove visitor classes (including `ExpressionContextVisitor`) out of `Main.java` and into their own files?